gladius_shared/src/settings.rs

```diff
@@ -209,6 +210,10 @@ pub struct Settings {
     #[AllowDefault]
     /// Settings for specific layers
     pub layer_settings: Vec<(LayerRange, PartialLayerSettings)>,
+
+    #[Optional]
+    /// Areas of the bed that can't have parts on it
+    pub bed_exclude_areas: Option<MultiPolygon>,
```

Comment:
Should be marked     
#[Combine]
#[AllowDefault]

this will allow multiple settings files to define different exclusion zones and then they are "union_with" each other.

A implimentation of combine would be something like

impl<T> Combine for MultiPolygon{
    fn combine(&mut self, other: Self) {
        self = self.union_with(&other);
    }
}

Reply:
Seems like a good idea